docs/faq.md

```diff
@@ -42,6 +42,27 @@ https://docs.docker.com/desktop/
 **A:**
 -->
 
+<!-- #640 -->
+**Q: How can I restart Rancher Desktop?**
```

Comment:
May be `How to gracefully shutdown Rancher Desktop and all the services underneath?`

also, `File -> Exit`, `Tray icon exit` are basic features available for desktop apps. So, do we want to keep this question in faqs?

Reply:
I agree that it's basic functionality, so I'll remove it!